functest/package_test.go

```diff
@@ -83,8 +83,40 @@ func TestPackageDockerOptions(t *testing.T) {
 	}
 }
 
-/*
-func TestPackageBuildahOptions(t *testing.T) {
+func TestPackageBuildah(t *testing.T) {
+	sandbox, cleanup := cmdtest.TestSetupWithSandbox(t, true)
+	defer cleanup()
+
+	var outBuffer bytes.Buffer
+	log := &cmd.LoggingConfig{}
+	log.InitLogging(&outBuffer, &outBuffer)
+
+	stackDir := filepath.Join(cmdtest.TestDirPath, "starter")
+	err := cmd.CopyDir(log, stackDir, sandbox.ProjectDir)
+	if err != nil {
+		t.Errorf("Problem copying %s to %s: %v", stackDir, sandbox.ProjectDir, err)
+	} else {
+		t.Logf("Copied %s to %s", stackDir, sandbox.ProjectDir)
+	}
+
+	// Because the 'starter' folder has been copied, the stack.yaml file will be in the 'starter'
+	// folder within the temp directory that has been generated for sandboxing purposes, rather than
+	// the usual core temp directory
+	sandbox.ProjectDir = filepath.Join(sandbox.ProjectDir, "starter")
+
+	args := []string{"stack", "package", "--buildah"}
+	output, err := cmdtest.RunAppsody(sandbox, args...)
+	if err != nil {
+		t.Fatal(err)
+	} else {
+		if !strings.Contains(output, "[Buildah] Writing manifest to image destination") {
+			t.Errorf("String \"[Buildah] Writing manifest to image destination\" not found in output")
```

Comment:
What exactly are we testing here? That outputs suggests that the command has passed, and the image is built?